core/src/main/java/io/seata/core/store/db/LogStoreDataBaseDAO.java

```diff
@@ -497,6 +513,63 @@ private BranchTransactionDO convertBranchTransactionDO(ResultSet rs) throws SQLE
         return branchTransactionDO;
     }
 
+    /**
+     * the public modifier only for test
+     */
+    public void initTransactionNameSize() {
+        Map<String, ColumnInfo> result = this.queryTableStructure(globalTable);
+        if (result == null || result.isEmpty()) {
+            LOGGER.warn("{} not found", globalTable);
+            return ;
+        }
+        ColumnInfo columnInfo = result.get(TRANSACTION_NAME_KEY);
+        int columnSize = columnInfo.getColumnSize();
+        this.transactionNameColumnSize = columnSize;
+    }
+
+    /**
+     * query column info from table
+     * @param table the table name
+     * @return the column info {`columnName upperCase`=columnInfo}
+     */
+    private Map<String, ColumnInfo> queryTableStructure(final String table) {
```

Comment:
reused?but private.  why not directly return ColumnInfo？

Reply:
I want to reuse it, but there is no place to use it.